packages/admin-ui-docs/src/components/IconsGrid/index.tsx

```diff
@@ -97,7 +97,7 @@ export function IconsGrid(props: IconsGridProps) {
         />
       </DataViewControls>
       {selectedWeight === 'Fill' || selectedSize === 'Small' ? (
-        <Alert visible csx={{ marginBottom: 4 }}>
+        <Alert csx={{ marginBottom: 4 }}>
```

Comment:
Why?

Reply:
this prop doesn't exist anymore, alert is always visible now